app/main.ipynb

```diff
@@ -2,17 +2,22 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 14,
```

Comment:
cannot check because of this issue. Here is how to fix it https://stackoverflow.com/questions/34085676/ipython-notebook-error-notebook-json-is-invalid-uexecution-count-is-a-requi
<img width="614" alt="image" src="https://github.com/mate-academy/py-restaurant-data-analysis/assets/61380547/1114774e-613b-4c10-8a72-caf9d8a8a1cf">